src/components/GameBoard.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import Cell from "./Cell";
+
+const width = 7, height = 6;
+
+export default () => {
+  const [isFinished, setIsFinished] = useState<boolean>(false);
+  const [winner, setWinner] = useState<string>("");
+  const [turn, setTurn] = useState<string>("Red");
+  const [boardData, setBoardData] = useState<Array<Array<string>>>([]);
+
+  const initBoard = () => {
+    let tempData = new Array(height);
+    for (let i = 0; i < height; i++) {
+      tempData[i] = new Array(width);
+      for (let j = 0; j < width; j++)
+        tempData[i][j] = "gray";
+    }
```

Comment:
Instead of using `for...let` clause, you can use `Array.map` to initiate the board.